src/main/java/io/pinecone/proto/VectorServiceOuterClass.java

```diff
@@ -0,0 +1,479 @@
+    internal_static_QueryResponse_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_QueryResponse_descriptor,
+        new String[] { "Results", "Matches", "Namespace", "Usage", });
+    internal_static_Usage_descriptor =
+      getDescriptor().getMessageTypes().get(18);
+    internal_static_Usage_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_Usage_descriptor,
+        new String[] { "ReadUnits", });
+    internal_static_UpdateRequest_descriptor =
+      getDescriptor().getMessageTypes().get(19);
+    internal_static_UpdateRequest_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_UpdateRequest_descriptor,
+        new String[] { "Id", "Values", "SparseValues", "SetMetadata", "Namespace", });
+    internal_static_UpdateResponse_descriptor =
+      getDescriptor().getMessageTypes().get(20);
+    internal_static_UpdateResponse_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_UpdateResponse_descriptor,
+        new String[] { });
+    internal_static_DescribeIndexStatsRequest_descriptor =
+      getDescriptor().getMessageTypes().get(21);
+    internal_static_DescribeIndexStatsRequest_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_DescribeIndexStatsRequest_descriptor,
+        new String[] { "Filter", });
+    internal_static_NamespaceSummary_descriptor =
+      getDescriptor().getMessageTypes().get(22);
+    internal_static_NamespaceSummary_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_NamespaceSummary_descriptor,
+        new String[] { "VectorCount", });
+    internal_static_DescribeIndexStatsResponse_descriptor =
+      getDescriptor().getMessageTypes().get(23);
+    internal_static_DescribeIndexStatsResponse_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_DescribeIndexStatsResponse_descriptor,
+        new String[] { "Namespaces", "Dimension", "IndexFullness", "TotalVectorCount", });
+    internal_static_DescribeIndexStatsResponse_NamespacesEntry_descriptor =
+      internal_static_DescribeIndexStatsResponse_descriptor.getNestedTypes().get(0);
+    internal_static_DescribeIndexStatsResponse_NamespacesEntry_fieldAccessorTable = new
+      com.google.protobuf.GeneratedMessageV3.FieldAccessorTable(
+        internal_static_DescribeIndexStatsResponse_NamespacesEntry_descriptor,
+        new String[] { "Key", "Value", });
+    com.google.protobuf.ExtensionRegistry registry =
+        com.google.protobuf.ExtensionRegistry.newInstance();
+    registry.add(com.google.api.FieldBehaviorProto.fieldBehavior);
+    registry.add(com.google.api.AnnotationsProto.http);
```

Comment:
Please add a `TODO` or similar comment here to keep track of places we've made changes to generated code.